test/test_subprocess.py

```diff
@@ -34,7 +34,7 @@ def test_timeout():
 sys.stdout.flush()
 sys.stderr.flush()
 subprocess.call([sys.executable, "-c",
-    "import sys, subprocess; subprocess.call([sys.executable, '-c', 'import time; time.sleep(360)'])"])
+    "import sys, subprocess; subprocess.call([sys.executable, '-c', 'import time; time.sleep(360)'])"])  # noqa E501 line too long
```

Comment:
I don't see any reason why this can't be splitted in multiple lines.

Reply:
I'm not sure why it is not easy to split this line. I tried a couple of things, and it doesn’t work:
Option 1:
<img width="1022" alt="image" src="https://user-images.githubusercontent.com/13194519/157081696-c38c1711-2e77-4a3c-860a-74e105cb59c4.png">
Option 2:
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/13194519/157085757-f8c8beec-450e-4f96-a025-8e86034ec60b.png">

Could you please help me?

Reply:
This is Python code as a string. It needs to be valid Python code when converted to a `.py` file. And this is clearly not, as you have a multiline string with single quotes.